.circleci/config.yml

```diff
@@ -0,0 +1,186 @@
+version: 2
+
+defaults: &defaults
+  working_directory: /go/src/github.com/weaveworks/scope
+  docker:
+    - image: weaveworks/scope-backend-build:circle-2.0-4ad2f14
+
+client-defaults: &client-defaults
+  working_directory: /home/weave/scope
+  docker:
+    - image: weaveworks/scope-ui-build:circle-2.0-2885645
```

Comment:
These two are on different versions, which seems odd?

Reply:
Once the changes are merged to master I will update to a master tag, the same tag for both images.